www/src/util/serverResponses.d.ts

```diff
@@ -0,0 +1,15 @@
+export type User = {
+    id: string;
+    email: string;
+    ccid: string;
+    program: string;
+    year: number;
+    given_name: string;
+    family_name: string;
+    full_name: string;
+    photo_url: string | null;
+    created_at: db.TimestampTzString;
+    updated_at: db.TimestampTzString;
+};
+
+// TODO: Find better way of syncing server reponse types
```

Comment:
Since the server already has types on the responses I thought there was a way to sync them up (say run zapatos on front-end to generate the types or use symlinks) but I think it's good that we decouple it like this just to avoid putting dependency to the back-end. I'll remove this TODO